src/main/java/racinggame/view/ScoreScreen.java

```diff
@@ -0,0 +1,39 @@
+package racinggame.view;
+
+import java.util.List;
+import java.util.Map;
+import racinggame.core.scoreboard.ScoreBoard;
+import racinggame.core.scoreboard.Score;
+
+public class ScoreScreen {
+
+    public static void printWinner(ScoreBoard scoreBoard) {
+        List<String> winners = scoreBoard.getWinner(
+                scoreBoard.getCountOfRound());
+        System.out.println(
+                String.format("%s가 최종 우승했습니다.",
+                        String.join(" ,",
+                                winners.toArray(new String[0]))
+                ));
```

Comment:
```suggestion
        System.out.printf(
                "%s가 최종 우승했습니다.%n",
                String.join(" ,", winners.toArray(new String[0]))
        );
```

`printRoundResult()` 역시 같은 방식으로 수정 가능해 보여요.